syntax="proto3";

package smollm;


service smollm {
    rpc sendReq (SmolLMReq) returns (stream SmolLMRes);
}

message SmolLMReq {
    string prompt=1;
}

message SmolLMRes {
    string curToken=1;
    uint32 tokenNum=2;
    float tokenProbability=3;
}